src/test/java/com/solvd/meta/carina/LoginTest.java

```diff
@@ -29,6 +29,7 @@ public void loginLockedUserTest() {
         Assert.assertTrue(loginPage.isPageOpened(), "Home page is not opened");
         loginPage.setUsername(R.TESTDATA.get("locked_user"));
         loginPage.setPassword(R.TESTDATA.get("standard_user_password"));
-        Assert.assertTrue(loginPage.isPageOpened(), "Login aproved after click login button");
+        InventoryPage inventoryPage = loginPage.clickLogIn();
```

Comment:
This assertion also can be here
`Assert.assertTrue(loginPage.isPageOpened(), "Login aproved after click login button");`